process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -446,6 +446,12 @@ func (st *storageBootstrapper) applyBlock(headerHash []byte, header data.HeaderH
 
 	st.blkc.SetCurrentBlockHeaderHash(headerHash)
 
+	sig, bitmap := header.GetPreviousAggregatedSignatureAndBitmap()
```

Comment:
updated as discussed: won't set the previous proof on storage bootrapper, but store it on the first received block after sync.. with this, the only situation a node won't have the proof is in case it is leader in the first round after sync. In this case, it will wait one more round.